src/snark/src/gadgetlib1/gadgets/merkle_tree/tests/test_merkle_tree_gadgets.cpp

```diff
@@ -29,6 +30,9 @@ TEST(gadgetlib1, merkle_tree)
 {
     start_profiling();
 
+    alt_bn128_pp::init_public_params();
+    test_all_merkle_tree_gadgets<alt_bn128_pp>();
+
 #ifdef CURVE_BN128       // BN128 has fancy dependencies so it may be disabled
```

Comment:
I wonder if we want to erase this, as we only use alt_bn128, not bn128.  @ebfull ?

Reply:
I wonder why in libsnark  https://github.com/scipr-lab/libsnark/issues/91
they do this test for all curves except alt_bn128? Just overlooked?

Reply:
I'm +0 on erasing this, because `src/Makefile.am` defines flags such that this won't cause us problems. If we want to strip this code out, we can do so in a follow-up PR.